builder/relinker/configuration.py

```
@@ -0,0 +1,301 @@
+        self.library = library
+        self.funcs = dict()
+        self.paths = paths
+        self.dumps = self.read_dump_info(paths)
+        self.section_all = False
+    
+    _WILDCARD_SECTIONS = {
+        '.text.*': '.literal .literal.* .text .text.*',
+        '.iram1.*': '.iram1 .iram1.*',
+        '.wifi0iram.*': '.wifi0iram .wifi0iram.*',
+        '.wifirxiram.*': '.wifirxiram .wifirxiram.*',
+    }
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)